modules/@apostrophecms/ui/ui/apos/components/AposFilterMenu.vue

```diff
@@ -65,9 +66,7 @@ export default {
           field: {
             name: filter.name,
             type: 'radio',
-            // TODO: Remove `|| filter.name.toUpperCase()` once filters include
```

Comment:
Should we support a more graceful automatic naming if label is missing? We do for schema fields. It's a bit of a parlor trick, but devs do appreciate that "save me all the time" stuff.

```
        if ((!field.label) && (!field.contextual)) {
          field.label = _.startCase(field.name.replace(/^_/, ''));
        }
```

Does a reasonably pretty job.

Reply:
At the moment it fails to have the filter label missing, but doesn't break the app. I'd be alright with defaulting to the `name` values. I'm hesitant to install a utility for this kind of thing. I'd rather make it clear that something needs fixing and how to fix it.

Reply:
Yeah, if we do it, it should be done server side where lodash is already available, and the configuration that reaches Vue should be ready to rock with a label. Which, puts it outside the scope of your PR. Approving.